.github/workflows/universalnrp-test-run.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     with:
       target: ${{ inputs.target }}
       arch: ${{ inputs.arch }}
-      artifact: ${{ inputs.target }}-${{ inputs.install-osconfig == true && 'withOSConfig' || 'noOSConfig' }}
+      artifact: ${{ inputs.target }}-${{ inputs.install-osconfig == true && 'withOSConfig' || 'withoutOSConfig' }}
```

Comment:
>withoutOSConfig [](http://example.com/codeflow?start=95&length=15)

Suggest recalling this PR. Wait for refactoring of the NRP etc. There will likely no 'withOSconfig' situation at all. No need to do changes back and forth. Instead, recall this PR and wait for that decision.

Reply:
Spoke offline, removed the tests with agent is present